src/database/models/reviews.js

```diff
@@ -44,7 +48,7 @@ module.exports = (sequelize, DataTypes) => {
 			indexes: [
 				{
 					unique: true,
-					fields: ['resource_id', 'reviewer_id'],
+					fields: ['resource_id', 'reviewer_id', 'organization_id'],
```

Comment:
@aks30 Org id we added in the unique index because of citus error. The primary key needs to add when define a unique index in citus. And also the resource I'd and the reviewer is unique across org.